doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -517,6 +517,14 @@ Multicore samples
 Networking samples
 ------------------
 
+* Removed:
+
+  * QEMU support.
+
+* Added:
+
+  * Native simulator support.
+
 * Updated:
 
   *  The networking samples to support import of certificates in valid PEM formats.
```

Comment:
```suggestion
* Updated the networking samples to support import of certificates in valid PEM formats.
*  Removed QEMU x86 emulation support and added support for the :ref:`native simulator <zephyr:native_sim>` board.
```
We could remove line no: 537 since we have mentioned about the support here for the networking samples in general.